src/app/[lang]/(home)/vocabulary/[word]/page.tsx

```diff
@@ -17,12 +17,15 @@ export default async function Vocabulary(props: VocabularyPageProps) {
 
   const dictionary = await getDictionary(params.lang)
 
-  const data = await VocabulariesService.getWordDetails(params.word)
+  const data = await VocabulariesService.getVocabs({
+    text: params.word,
+    is_strict: true,
+  })
 
-  if (data.isError) {
+  if (data.isError || data?.data?.length === 0) {
```

Comment:
![Screenshot 2024-11-24 at 21 51 58](https://github.com/user-attachments/assets/ea9c6bb5-1ca0-4e52-b9a9-95f9d22725ce)

Please make sure the data is available, Don't abuse the structure?.